services/topology-engine/queue-engine/topologylistener/message_utils.py

```diff
@@ -189,15 +189,18 @@ def build_one_switch_flow_from_db(switch, stored_flow, output_action):
     return flow
 
 
-def build_delete_flow(switch, flow_id, cookie, meter_id, in_port, in_vlan, out_port):
+def build_delete_flow(node):
```

Comment:
`node` bad name... there is no associations for it in our domain model. It should be  something like `flow_segments` or `path_segment`.